Day5/style.css

```diff
@@ -0,0 +1,13 @@
+#color-field {
+  width: 200px;
+  height: 200px;
+  background-color: rgb(0,0,0);
+  border-radius: 50%;
```

Comment:
The `%` here is fine, because the `width` and `height` of this element is stable.
This `%` right here is equal to `border-radius: 100px; // (width/2 => 100px)`